src/miniscript/types/correctness.rs

```diff
@@ -164,7 +165,7 @@ impl Property for Correctness {
         }
     }
 
-    fn from_multi(_: usize, _: usize) -> Self {
+    fn from_multi<Ctx: ScriptContext>(_: usize, _: usize) -> Self {
```

Comment:
Why do we need `ctx` here? multi should have SigType `Ecdsa` and `multi_a` should have `Schnorr`

Reply:
I think this would also resolve the awkward things that you had to do later with the `0` value

Reply:
Initially I introduced this to keep a check on misuse of SigType, but that is indeed taken care of in the consensus checks. Will update that in the following commit!